### ISSUE TITLE
Redundant table rows 

### ISSUE THREAD

**Issue by chcurlet-msft** [MicrosoftDocs/windows-uwp#2034]
The 3rd and 4th rows of the "Detailed listing of desktop behavior" table are redundant ("Battery Saver enabled" and "On battery + battery saver enabled + screen off").

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 00495cf6-5df2-c85a-be4b-5af5ffb95acb
* Version Independent ID: 9234cef0-7a0d-7d2d-ef53-ed1df59ece26
* Content: [WNS Notification Priorities - Windows UWP applications](https://docs.microsoft.com/en-us/windows/uwp/design/shell/tiles-and-notifications/wns-notification-priorities#feedback)
* Content Source: [windows-apps-src/design/shell/tiles-and-notifications/WNS-Notification-Priorities.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/windows-apps-src/design/shell/tiles-and-notifications/WNS-Notification-Priorities.md)
* Product: **uwp**
* Technology: **design**
* GitHub Login: @hickeys
* Microsoft Alias: **hickeys**

**Comment by hickeys** [comment]
This is true, but from the article: "OEMs are free to configure the behavior differently, but most are close to this chart." I think it makes sense to leave each state on a separate line to emphasize that, though the defaults are the same, they are technically different states and could potentially behave differently.

I'm open to good reasons why those rows should be collapsed into a single row. Otherwise, I'll close this issue in a few days.

**Comment by chcurlet-msft** [comment]
By that argument, all different permutations of screen off/on, Battery Saver off/on, and AC/DC power could be different rows. I'd reason that there should be minimal rows in order to fully cover all states' differences in behavior, where the two redundant rows could then be collapsed into one.

**Comment by GrantMeStrength** [comment]
Closing — this issue references a UWP page (`windows-apps-src/`) that is now maintained in the [windows-uwp](https://github.com/MicrosoftDocs/windows-uwp) repository, not this one. If the issue is still relevant, please re-file it there. Thank you!
